.goreleaser.yml

```diff
@@ -71,3 +71,11 @@ changelog:
     exclude:
     - '^docs:'
     - '^test:'
+release:
+  footer: |
+    ### Container Image
+
+    ```
+    docker pull gcr.io/go-containerregistry/crane:{{.Tag}}
+    docker pull gcr.io/go-containerregistry/gcrane:{{.Tag}}
```

Comment:
I don't find a better way of doing it, though - This change, as is, is still helpful!

Reply:
Good catch @zhouhaibing089! I'll wait for https://github.com/goreleaser/goreleaser/pull/2326 to merge this. If it takes too long I'll update to not include the `{{.Tag}}` since that's better than nothing.

Reply:
The referenced PR has now been merged. :)